app/views/widgets/home_page/_general_info.html

```diff
@@ -1,7 +1,6 @@
 <section class="section light-bg" id="generalInformation">
   <div class="maxWidth">
-    <h2 class="heading">At Chapman, you'll discover there's an</h2>
-    <h3 class="subheading" data--150-bottom="opacity:1;" data--150-bottom-top="opacity:0;">energy that runs through everything and everyone.</h3>
+    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">At Chapman, you'll discover there's an energy that runs through everything and everyone.</h2>
```

Comment:
Looks like another extra quotation mark. Slightly surprised this isn't breaking.